packages/-ember-data/tests/integration/adapter/store-adapter-test.js

```diff
@@ -1169,7 +1169,7 @@ module('integration/adapter/store-adapter - DS.Store and DS.Adapter integration
               },
             },
           });
-          assert.ok(tom.get('dogs') instanceof PromiseArray, 'dogs is a promise');
+          assert.ok(typeof tom.dogs.then === 'function', 'dogs is a thenable');
```

Comment:
Completely fine if not, but any way to also assert that it's an array?

Reply:
not really, since it wont one. We can assert it's an instance of our class still but the main thing is that its a thenable so I left it at that.